static/docs/user-guide/experiments/tags.md

```diff
@@ -0,0 +1,115 @@
+# How to Manage Experiments by Tags
+
+You can mark (or label) each experiment using Git tags.
+
+<p align="center">
+<img src="/static/img/user-guide/experiments/tags.png" />
+</p>
+
+Using tags to mark experiments is most suitable when you can easily create a new
+experiment based on the last one. For example when you are trying different
+values for only one aspect of the experiment, say feature extraction.
+
+## Examples
+
+Examples of managing experiments by tags can be seen on these sections of the
+Get Started tutorial:
+
+- [Metrics](/doc/get-started/metrics)
+- [Experiments](/doc/get-started/experiments)
+- [Compare Experiments](/doc/get-started/compare-experiments)
+- [Get Older Files](/doc/get-started/older-versions)
+
+They can also be tried interactively on Katacoda:
+
+- [Experiments and Metrics](https://katacoda.com/dvc/courses/get-started/experiments)
+- [Data Versioning](https://katacoda.com/dvc/courses/get-started/versioning)
+
+## How it works
+
+### Commit and tag
+
+At the end of each experiment we commit everything to Git and add a tag, like
+this:
+
+```dvc
+$ git add .
+$ git commit -m 'Baseline experiment'
+$ dvc commit
+$ git tag baseline-experiment
```

Comment:
it is a lightweight tag. Why do we use it here as opposed to the regular one below? Are you sure that metrics show works well with it?

Reply:
This one is a "normal" tag, the other one is an annotated tag. It is just an example.
As far as I know `metrics show` works with all the tags.

Reply:
it's not even about what can be called normal. It's more about - why do we use a lightweight here and an annotated below? If there are some reasons then explanation is required when should I use what type.

Reply:
I don't think there is any reason. They are both tags and both of them can be used.

Reply:
so, why use both? let's simplify. Though, there are some guidelines behind those. I think people tend to use lightweight tags for things that they won't be sharing/saving. May be we can use that difference as a guideline? Not sure that we need put so many details - I would probably just keep simple version in both cases if it works fine.